app/src/androidTest/java/com/github/houseorganizer/houseorganizer/FirebaseTestsHelper.java

```diff
@@ -191,4 +210,44 @@ protected static void setUpFirebase() throws ExecutionException, InterruptedExce
 
```

Comment:
Does create task list require user_1 to be logged in ? If not could you move it above the sign in ?

Reply:
Sure, I can move it - it doesn't require a user for now